10/jessie/Dockerfile

```diff
@@ -61,4 +61,8 @@ RUN set -ex \
   && ln -s /opt/yarn-v$YARN_VERSION/bin/yarnpkg /usr/local/bin/yarnpkg \
   && rm yarn-v$YARN_VERSION.tar.gz.asc yarn-v$YARN_VERSION.tar.gz
 
+COPY docker-entrypoint.sh /usr/local/bin/
+RUN ln -s usr/local/bin/docker-entrypoint.sh / # backwards compat
```

Comment:
You've never had an entrypoint before so this symlink is unnecessary.

Reply:
AAh, I wasn't sure if it was compatibility with older versions of docker. Good to know!

Reply:
> ใช่ฉันไม่แน่ใจว่ามันเข้ากันได้กับ Docker รุ่นเก่าหรือไม่ ดีแล้วที่รู้!